obsreport/obsreport_exporter.go

```diff
@@ -101,13 +102,9 @@ func StartTraceDataExportOp(
 func EndTraceDataExportOp(
 	exporterCtx context.Context,
 	numExportedSpans int,
-	numDroppedSpans int, // TODO: For legacy measurements, to be removed in the future.
+	numDroppedSpans int,
```

Comment:
Should this be removed? I am curios why is this not used for new metric. Seems like a useful thing.

Reply:
Good catch, I was playing a bit with this and forgot to remove, thanks